### PR TITLE
Fix part of #2833: Added named args in functions of TopicRevisionFragmentTest

### LINKED ISSUE #2833: Add missing named argument
**Describe the bug**
We are passing arguments without specifying their names.

**Expected behavior**
Look for the below-mentioned test files and add all the missing named arguments 

- [x] AdministratorControlsActivityTest (@dishantvyas15 )
- [x] AppVersionActivityTest (@darshanhande11)
- [x] CompletedStoryListActivityTest (@techjd)
- [x] FAQListFragmentTest (@ksog66)
- [x] HelpFragmentTest (@ravi9607)
- [x] RecentlyPlayedFragmentTest (@techjd)
- [x] OngoingTopicListActivityTest (@nlok5923)
- [x] OptionsFragmentTest  (@yashyon)
- [x] ProfileChooserFragmentTest (@mrwhoknows55)
- [x] ProfileProgressFragmentTest (@Arnold2381)
- [x] BindableAdapterTest (@jkugsiya)
- [x] ProfileEditActivityTest (@ArpitShukIa)
- [x] ProfileListFragmentTest (@Naman-1234)
- [x] SplashActivityTest (@darshanhande11)
- [x] StoryActivityTest (@jkugsiya)
- [x] StoryFragmentTest (@ChiragKalra)
- [x] DragDropTestActivityTest (@sakshampruthi)
- [x] ImageRegionSelectionInteractionViewTest (@Karanjot-singh)
- [x] TopicTestActivityForStoryTest (@nlok5923)
- [x] TopicInfoFragmentTest (@darshanhande11)
- [x] TopicLessonsFragmentTest (@Amar-2003)
- [x] TopicPracticeFragmentTest (@Karanjot-singh)
- [x] TopicRevisionFragmentTest (@justdvnsh)

**Additional context**
What is Named Arguments? 
https://kotlinlang.org/docs/functions.html#named-arguments

**How to work on this issue**
Comment below "I want to work on [FILE_NAME]", mention the file name from the above list you want to work on. 
Please work on only one file at a time. 

If you are a First Time Contributor to Oppia-Android, Start [here](https://github.com/oppia/oppia-android/wiki#onboarding-instructions).

**Note: Don't assign this issue to anyone as this issue should be done in a multiple PR fashion "single file, single PR"**

### PR DESCRIPTION
**Pull request by justdvnsh**
## Explanation
Fix part of  #2833 . Added missing named arguments in the functions of TopicRevisionFragmentTest

## Checklist
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt [hunk 1]
```diff
@@ -112,36 +112,58 @@ class TopicRevisionFragmentTest {
   }
 
   private fun setUpTestApplicationComponent() {
-    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
+    ApplicationProvider.getApplicationContext<TestApplication>().inject(
+      topicRevisionFragmentTest = this
```
Comment (prayutsu) [inline app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt]: We do not require a named argument here as there's only a single argument here.

File: app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt [hunk 2]
```diff
@@ -180,21 +214,25 @@ class TopicRevisionFragmentTest {
...
+    return launch(
+      createTopicActivityIntent(internalProfileId = internalProfileId, topicId = topicId)
+    )
   }
 
   private fun clickRevisionTab() {
     onView(
       allOf(
-        withText(TopicTab.getTabForPosition(3).name),
+        withText(TopicTab.getTabForPosition(position = 3).name),
```
Comment (prayutsu) [inline app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt:233]: Ditto here.

File: app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt [hunk 3]
```diff
@@ -241,16 +279,18 @@ class TopicRevisionFragmentTest {
   class TestApplication : Application(), ActivityComponentFactory, ApplicationInjectorProvider {
     private val component: TestApplicationComponent by lazy {
       DaggerTopicRevisionFragmentTest_TestApplicationComponent.builder()
-        .setApplication(this)
+        .setApplication(application = this)
```
Comment (prayutsu) [inline app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt]: Ditto here.

File: app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt [hunk 4]
```diff
@@ -241,16 +279,18 @@ class TopicRevisionFragmentTest {
...
         .build() as TestApplicationComponent
     }
 
     fun inject(topicRevisionFragmentTest: TopicRevisionFragmentTest) {
-      component.inject(topicRevisionFragmentTest)
+      component.inject(topicRevisionFragmentTest = topicRevisionFragmentTest)
     }
```
Comment (prayutsu) [inline app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt]: Ditto here.

File: app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt [hunk 5]
```diff
@@ -241,16 +279,18 @@ class TopicRevisionFragmentTest {
...
 
     override fun createActivityComponent(activity: AppCompatActivity): ActivityComponent {
-      return component.getActivityComponentBuilderProvider().get().setActivity(activity).build()
+      return component.getActivityComponentBuilderProvider().get().setActivity(
+        appCompatActivity = activity
+      ).build()
```
Comment (prayutsu) [inline app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt]: Ditto here.

**Review by prayutsu — CHANGES_REQUESTED** [review]
@justdvnsh PTAL, suggest some nit fixes.

**Comment by prayutsu** [comment]
@justdvnsh A few quick tips for making a good PR - 
1. The branch name should be descriptive and different words should be separated by hyphens (No other characters should be used except lowercase English alphabet and '-').
2. The commit messages should never include keywords like `Fixes #bugNum` because it leads to unnecessary mentions of the commit in the issue description.
3. When PR is ready for review, add a "PTAL" comment by mentioning any reviewers like - `@{reviewerName} PTAL.` This comment ensures that PR is ready to review from your side (if the author does not leave a PTAL comment then the reviewer may think that the author wants to add more commits in the PR)

I suggest going through the [Making a Code Change Instructions](https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change) once.

File: app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt (hunk 2 above)
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt:233]: @prayutsu We do require a named argument here as simply passing `3` sometime not makes the code readable as we cannot simply understand what is the number meaning we are passing as an argument. So, we do require this named argument here.

Comment (prayutsu) [reply app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt:233]: @anandwana001 Thanks for clarifying.
@justdvnsh you can ignore this and resolve this particular conversation.

File: app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt (hunk 1 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt]: +1

File: app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt (hunk 3 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt]: +1

File: app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt (hunk 4 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt]: +1

File: app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt (hunk 5 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt]: +1

**Review by rt4914 — CHANGES_REQUESTED** [review]
@justdvnsh Please make changes as suggested by @prayutsu . Also, once solved do reply to all the comments.

Thanks.

**Comment by justdvnsh** [comment]
@prayutsu PTAL.

File: app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt (hunk 1 above)
Comment (justdvnsh) [reply app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt]: PTAL

File: app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt (hunk 3 above)
Comment (justdvnsh) [reply app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt]: PTAL

File: app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt (hunk 4 above)
Comment (justdvnsh) [reply app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt]: PTAL

File: app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt (hunk 5 above)
Comment (justdvnsh) [reply app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt]: PTAL

**Comment by justdvnsh** [comment]
> 
> 
> @justdvnsh A few quick tips for making a good PR -
> 
>     1. The branch name should be descriptive and different words should be separated by hyphens (No other characters should be used except lowercase English alphabet and '-').
> 
>     2. The commit messages should never include keywords like `Fixes #bugNum` because it leads to unnecessary mentions of the commit in the issue description.
> 
>     3. When PR is ready for review, add a "PTAL" comment by mentioning any reviewers like - `@{reviewerName} PTAL.` This comment ensures that PR is ready to review from your side (if the author does not leave a PTAL comment then the reviewer may think that the author wants to add more commits in the PR)
> 
> 
> I suggest going through the [Making a Code Change Instructions](https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change) once.

@prayutsu I'll definitely keep that in mind

**Review by prayutsu — APPROVED** [review]
LGTM, Thanks @justdvnsh !!
Feel free to resolve all the conversations initiated by me.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**